Assets/Tests/Common/AsyncUtil.cs

```diff
@@ -11,7 +11,12 @@ public static class AsyncUtil
 
         public static UniTask WaitUntilWithTimeout(Func<bool> predicate)
         {
-            return UniTask.WaitUntil(predicate).Timeout(TimeSpan.FromSeconds(2));
+            return WaitUntilWithTimeout(predicate, 2);
+        }
+
+        public static UniTask WaitUntilWithTimeout(Func<bool> predicate, double seconds)
```

Comment:
might be better to have default arg with `seconds=2` instead of 2 functions

Reply:
you right. opps forgot that was a thing i guess lol